blankiety_galantow/core/room.py

```diff
@@ -22,10 +23,21 @@ def __init__(self, name):
         self.settings.name = name
         self.admin = None
 
+    def __del__(self):
+        self.timer_cancel()
+
     @property
     def number_of_players(self):
         return len(self.players)
 
+    @property
+    def is_empty(self):
+        if self.number_of_players == 0 :
+            return True
+        else:
+             return False
+
```

Comment:
Nie trzeba tutaj dodatkowych branchy w kodzie. KISS - Keep It Simple Sir 😉 
```suggestion
    @property
    def is_empty(self):
        return self.number_of_players == 0
```